back/engines/commercial/idea_assignment/spec/services/side_fx_idea_spec.rb

```diff
@@ -54,22 +54,6 @@
         expect { service.before_update(idea, nil) }.not_to change(idea, :assignee)
       end
     end
-
-    it 'sets the assignee to the default_assignee of the project on publish' do
-      default_assignee = create(:admin)
-      project = create(:project, default_assignee: default_assignee)
-      idea = create(:idea, project: project, publication_status: 'draft')
-      idea.publication_status = 'published'
-      expect { service.before_update(idea, user) }.to change { idea.assignee }.from(nil).to(default_assignee)
-    end
-
-    it "doesn't set the assignee if it's already set before publish" do
-      default_assignee = create(:admin)
-      project = create(:project, default_assignee: default_assignee)
-      idea = create(:idea, project: project, publication_status: 'draft', assignee: create(:admin))
-      idea.publication_status = 'published'
-      expect { service.before_update(idea, user) }.not_to(change { idea.assignee })
-    end
```

Comment:
It seems we already have tests of assignee assignment in `before_create` in this spec file, [here](https://github.com/CitizenLabDotCo/citizenlab/blob/2e6b4f171021c845918ab9f07d1dbb5966292b54/back/engines/commercial/idea_assignment/spec/services/side_fx_idea_spec.rb#L9-L26).